src/client.ts

```diff
@@ -641,6 +648,7 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
    * @returns a promise that fullfils once the session reject goes out
    */
   async rejectPendingSession (params: IPendingSessionActionParams): Promise<void> {
+    //This seems to not need any extra work as previously thought; keep eye on though
```

Comment:
can we remove this comment. in a month we are going to have no idea what this means.